package.json

```diff
@@ -35,13 +35,41 @@
   "main": "./out/extension.js",
   "l10n": "./l10n",
   "contributes": {
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+          "when": "sfdx_project_opened"
+        }
+      ]
+    },
     "commands": [
       {
         "command": "salesforcedx-vscode-offline-app.onboardingWizard",
         "title": "%extension.commands.config-wizard.title%",
         "category": "%extension.commands.config-wizard.category%"
+      },
+      {
+        "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+        "title": "%extension.commands.config-linting-tools.title%",
+        "category": "%extension.commands.salesforce-mobile-offline.category%"
+      }
+    ],
+    "configuration": {
+      "title": "%salesforce.mobile.extensions%",
+      "properties": {
+        "mobileOfflineLinting.eslint-plugin-lwc-graph-analyzer": {
```

Comment:
Without preceding `mobileOfflineLinting` separated by a dot the configuration doesn't get rendered in the settings page of VSCode. With it it becomes category name separated by a colon.

![image](https://github.com/salesforce/salesforcedx-vscode-mobile/assets/42476999/2d89db8d-4872-4e00-ac08-a87aa7b5fa69)